tool/src/org/antlr/v4/tool/Grammar.java

```diff
@@ -74,21 +74,28 @@
 	 */
 	public static final String INVALID_RULE_NAME = "<invalid>";
 
-	public static final Set<String> parserOptions = new HashSet<String>();
+	public static final String caseInsensitiveOptionName = "caseInsensitive";
+
+	public static final Set<String> ParserOptions = new HashSet<String>();
```

Comment:
I think for consistency we should keep this as initial lowercase, just like the others.

Reply:
I've encountered a mess with lower/UPPER case identifiers: https://github.com/antlr/antlr4/blob/master/tool/src/org/antlr/v4/tool/Grammar.java#L89-L95 So, you think it's better to use lower case for all options, do you?

Reply:
well I reserve the first capital letter for types like classes... at the very least we should be consistent at this point